wallet/wallet.go

```diff
@@ -1165,7 +1189,50 @@ func (w *Wallet) PurchaseTicketsContext(ctx context.Context, n NetworkBackend, r
 	}
 	defer heldUnlock.release()
 
-	return w.purchaseTickets(ctx, op, n, req)
+	ticketsTx, _, err := w.purchaseTickets(ctx, op, n, req)
+	if err != nil {
+		return nil, errors.E(op, err)
+	}
+
+	ticketsHashes, err := ticketsTxToHash(ticketsTx)
+	if err != nil {
+		return nil, errors.E(op, err)
+	}
+
+	return ticketsHashes, nil
+}
+
+// CreateUnsignedTickets creates unsigned tickets, returning its
+// hex transactions.
+func (w *Wallet) CreateUnsignedTickets(minBalance, spendLimit dcrutil.Amount, minConf int32, votingAddr dcrutil.Address, account uint32, count int, poolAddress dcrutil.Address,
+	poolFees float64, expiry int32, txFee dcrutil.Amount, ticketFee dcrutil.Amount) ([]*wire.MsgTx, *txauthor.AuthoredTx, error) {
```

Comment:
I'd rather not add any new wallet methods that take all these parameters.  We can instead change PurchaseTicketsRequest to export the fields that you need to set for VSP-bought tickets (and might as well remove the old PurchaseTickets and rename PurchaseTicketsContext to it instead, since we haven't stabilized wallet/v3 yet).